easybuild/easyconfigs/r/RStan/RStan-2.21.5-foss-2021b-R-4.1.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Bundle'
+
+name = 'RStan'
+version = '2.21.5'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/stan-dev/rstan'
+description = """RStan is the R interface to Stan. Stan is a state-of-the-art
+ platform for statistical modeling and high-performance statistical computation."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('R', '4.1.2')
+]
+
+# preinstallopts = 'export PKG_CXXFLAGS="$CXXFLAGS" && '
+# preinstallopts = 'export PKG_CXX14FLAGS="$CXX14FLAGS -DSTAN_THREADS" &&'
+
```

Comment:
```suggestion
```

We should remove the commented out code.

Reply:
OK, I'll remove those lines.